src/Apps/Auction/Components/AuctionFAQ.tsx

```diff
@@ -0,0 +1,158 @@
+import { Box, media, Serif, space, Spacer, Theme, Toggle } from "@artsy/palette"
+import { AuctionFAQ_viewer } from "__generated__/AuctionFAQ_viewer.graphql"
+import { AuctionFAQQuery } from "__generated__/AuctionFAQQuery.graphql"
+import { useSystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React from "react"
+import Markdown from "react-markdown"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components"
+
+interface AuctionFAQProps {
+  viewer: AuctionFAQ_viewer
+}
+
+export const AuctionFAQ: React.FC<AuctionFAQProps> = ({ viewer }) => {
```

Comment:
Our old approach for `multi-page` components like this FAQ was to have a [config file](https://github.com/artsy/force/blob/master/src/desktop/components/multi_page/config.coffee) that specified all of the necessary metadata for rendering these.

It seems like we could build something similar here instead of making this specific to auctions. I'm curious if you think going down that path (a single place to define the necessary data, with a generic component that could work for any of these) would be worthwhile?

We'll be building out the Collector FAQs next, so it would be good to make sure there aren't any limitations to that approach at this point, even if we do the refactor as part of that effort.

Reply:
This is such a simple component I think even if we had to refactor it would be trivial. I'd rather not add anymore complexity to this for now.